### PR TITLE
chore: hookup staging Gov Pay for Camden

### PR DESCRIPTION
**Pull request by jessicamcinchak**
Was originally hoping to do this directly via #2790, but Ian chased for an update this afternoon so doing the old fashioned way to get to staging for testing :+1: 

pizza-secrets additionally updated on S3

### CODE REVIEW

**Comment by jessicamcinchak** [comment]
[Hook up Gov Pay for Camden](https://trello.com/c/QcYdOgfK/2794-hook-up-gov-pay-for-camden)

File: docker-compose.yml [hunk 1]
```diff
@@ -155,6 +155,8 @@ services:
       UNIFORM_CLIENT_AYLESBURY_VALE: ${UNIFORM_CLIENT_AYLESBURY_VALE}
       UNIFORM_CLIENT_CHILTERN: ${UNIFORM_CLIENT_CHILTERN}
       UNIFORM_CLIENT_WYCOMBE: ${UNIFORM_CLIENT_WYCOMBE}
+      #Camden
+      GOV_UK_PAY_TOEKN_CAMDEN: ${GOV_UK_PAY_TOKEN_CAMDEN}
```
Comment (DafyddLlyr) [inline docker-compose.yml:159]:
```suggestion
      GOV_UK_PAY_TOKEN_CAMDEN: ${GOV_UK_PAY_TOKEN_CAMDEN}
```

**Review by DafyddLlyr — CHANGES_REQUESTED** [review]
One small typo!

Thanks for trying to hold this one off - agree that moving it forward is the right approach. I'll encrypt and set up the value and make a note of this on the other PR 👍

**Comment by jessicamcinchak** [comment]
Typo fixed, thanks !

**Pull request merged**